vnet/bin/vnflows-monitor

```diff
@@ -464,57 +442,30 @@ def output_flows(outputflows)
     
     puts flow.to_pretty
     linecount = 1 + linecount
-    break if ( $opts.head > 0 ) && ( linecount > $opts.head )
+    break if ( $opts[:head] > 0 ) && ( linecount > $opts[:head] )
   }
   puts "-------run:#{$run_n}--iteration:#{$inputcount}-------"
   $count = $count + 1  
-  exit if ( $opts.maxcount > 0 ) && ( $count >= $opts.maxcount )
+  exit if ( $opts[:count] > 0 ) && ( $count >= $opts[:count] )
 end
 
 def do_sleep
-  sleep $opts.interval
+  sleep $opts[:time]
 end
 
 def update_options
   while $stdin.ready?
-    $opts.parse_options $stdin.readline
+    $stdin.readline
   end
```

Comment:
The idea here was to make the tool more interactive.  For example, you could type "t 1 <ret>" while the program was running and just change the time delay without restarting.  If there were various levels of pretty printing implemented you could change them on-the-fly while debugging.  Maybe there is no easy way to keep this feature with OpenParser, in which case you should just delete the update_options method.